.buildkite/hooks/pre-command

```diff
@@ -67,4 +67,9 @@ mkdir -p "$JAVA_HOME"
 tar --extract --file /tmp/jdk.tar.gz --directory "$JAVA_HOME" --strip-components 1
 
 export JAVA_HOME
-export PATH=$JAVA_HOME/bin:$PATH
+PATH=$JAVA_HOME/bin:$PATH
+export PATH
```

Comment:
shellint is the one linting this!  https://github.com/buildkite/repo-hook-example/blob/7099120f40b81a979f3cbe665166bcee1d887eea/.buildkite/hooks/pre-command#LL7C68-L7C118 also uses it...